tests/product/test_server_install.py

```diff
@@ -335,10 +336,11 @@ def test_install_interactive_with_ips(self):
         self.copy_presto_rpm_to_master()
 
         cmd_output = self.run_prestoadmin_script(
-            "echo -e 'root\n22\n%s\n%s\n' | "
-            "./presto-admin server install /mnt/presto-admin/%s " %
+            'echo -e \'root\n22\n%s\n%s\n\' | '
```

Comment:
I think when quotes are nested we should alternate between double and single. For a string by itself, I've been converting everything to single quotes. They look neater to me.